pkg/etcd/etcd.go

```diff
@@ -796,6 +818,7 @@ func (e *ETCD) cluster(ctx context.Context, forceNew bool, options executor.Init
 			ClientCertAuth: true,
 			TrustedCAFile:  e.config.Runtime.ETCDPeerCA,
 		},
+		SnapshotCount:                   10000,
```

Comment:
This is for consistency with kubeadm et al:
https://github.com/etcd-io/etcd/issues/13889